apps/frontend/src/app/(protect-routes)/posts/[slug]/page.tsx

```
@@ -0,0 +1,168 @@
+            </Link>
+
+            <SocialActions post={post} size="lg" />
+          </div>
+        </header>
+
+        {/* Content */}
+        <Card>
+          <CardContent className="p-8">
+            <div
+              className="prose prose-lg max-w-none"
+              dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)